envoy/api/v2/listener/listener.proto

```diff
@@ -110,10 +112,10 @@ message FilterChain {
   google.protobuf.BoolValue use_proxy_proto = 4;
 
   // [#not-implemented-hide:] filter chain metadata.
-  Metadata metadata = 5;
+  core.Metadata metadata = 5;
 
   // [#not-implemented-hide:] See base.TransportSocket description.
```

Comment:
Can you update the doc to match what Cluster has? Also remove `not-implement-hide`. Thanks!

Reply:
Done PTAL

Reply:
? I don't see this is updated.

Reply:
Oh, I thought you are referring to Metadata.

Reply:
Can you give more details what needs to go here?

Reply:
Just like clusters

```
See :ref:`base.TransportSocket<envoy_api_msg_core.TransportSocket>` description.
```
Is good, no longer need `[#not-implemented-hide:]`

Reply:
Done